install-deps.sh

```diff
@@ -19,12 +19,52 @@ if test $(id -u) != 0 ; then
 fi
 export LC_ALL=C # the following is vulnerable to i18n
 
+function ensure_decent_gcc_on_deb {
+    # point gcc to the one offered by distro if the used one is different
+    local old=$(readlink -e /usr/bin/x86_64-linux-gnu-gcc | cut -d'-' -f2)
```

Comment:
@tchaikov Shaman builds of "jewel" are now failing - https://jenkins.ceph.com/job/ceph-dev-setup/14285//consoleFull

```
dpkg: warning: version 'linux' has bad syntax: version number does not start with digit
update-alternatives: error: alternative path /usr/bin/gcc-linux doesn't exist
Build step 'Execute shell' marked build as failure
```

Is it possible that `/usr/bin/x86_64-linux-gnu-gcc` points to `/usr/bin/gcc-linux`?

Reply:
@smithfarm #19521.